samples/c/common/opencv_c_wrapper/bmp_reader.c

```diff
@@ -70,12 +70,14 @@ int readBmpImage(const char* fileName, BitMap* image) {
 
     if (image->infoHeader.bits != 24) {
         printf("[BMP] 24bpp only supported. But input has: %d\n", image->infoHeader.bits);
-        return 3;
+        status = 3;
```

Comment:
Don't we want to have enum (with 3,4,5, etc. values) for code of status?

Reply:
I think no, It doesn't make sense and makes the code more difficult. We have output message for it.